execution_engine_testing/test_support/CHANGELOG.md

```diff
@@ -15,11 +15,13 @@ All notable changes to this project will be documented in this file.  The format
 
 ### Added
 * Added `WasmTestBuilder::get_execution_journals` method for returning execution journals for all test runs.
+* Added support to load values from a given Chainspec.
+* Added static and constants that represent Casper-mainnet chainspec values
```

Comment:
It's worth noting that chainspec might change with new protocol versions. We should include that in this changelog entry and specify which version of the chainspec it adds.

Reply:
Applied suggestion